### ISSUE TITLE
Outline view 'waiting for cargo metadata' not functioning due to rust-analyzer packaging issue.

### ISSUE THREAD

**Issue by hydra** [LucasBullen/redOx#410]
TL:DR

The 'download rust analyzer' button in the 'Rust' preferences page downloads an artifact 'rust-analyzer-x86_64-pc-windows-msvc.gz' that contains an executable file, 'rust-analyzer' which does not have the '.exe' extension on it and Eclipse seems unable to run it, thus the outline view does not work for rust files.

Full details of the issue are here, including a temporary workaround:

https://github.com/rust-lang/rust-analyzer/issues/10910#issuecomment-1225823218

I'm creating this issue so that if changes need to be made in Eclipse Corrosion for windows users so that it correctly uses the `.exe` extension for the rust-analyzer executable then the changes can be tracked here.

```
  Corrosion: Rust edition in Eclipse IDE	1.2.4.202206282034	org.eclipse.corrosion.feature.feature.group	Eclipse Corrosion
```

**Comment by hydra** [comment]
Also related: https://github.com/eclipse/corrosion/issues/394
